src/main/logging/README.md

```diff
@@ -0,0 +1,244 @@
+
+To start first enable `fluentd` and set the `hostname` for elasticsearch in volues.yaml as follows:
+
+```yaml
+fluentd:
+   enabled: true
+   elasticsearch:
+     hostname: elasticsearch-master
+```
+
+Now fluentd tries to parse and send the data to elasticsearch but as it is not installed yet then data get lost. At this point we should install elasticsearch and kibana to complete EFK stack deployment.
+
+```shell script
+$ helm repo add elastic https://helm.elastic.co
+"elastic" has been added to your repositories
+
+$ helm install elasticsearch elastic/elasticsearch
+...
+```
+
+wait until all nodes get up and running status:
+
+```shell script
+$ kubectl get pods --namespace=dcd -l app=elasticsearch-master
+NAME                     READY   STATUS    RESTARTS   AGE
+elasticsearch-master-0   1/1     Running   0          100m
+elasticsearch-master-1   1/1     Running   0          100m
+elasticsearch-master-2   1/1     Running   0          100m
+$
+$ kubectl port-forward svc/elasticsearch-master 9200
+```
+make sure elasticsearch cluster working as expected:
+```shell script
+$ curl localhost:9200
+{
+  "name" : "elasticsearch-master-0",
+  "cluster_name" : "elasticsearch",
+  "cluster_uuid" : "uNdYC-2nSdWVdzPCw9P7jQ",
+  "version" : {
+       "number" : "7.12.0",
+       "build_flavor" : "default",
+       "build_type" : "docker",
+       "build_hash" : "78722783c38caa25a70982b5b042074cde5d3b3a",
+       "build_date" : "2021-03-18T06:17:15.410153305Z",
+       "build_snapshot" : false,
+       "lucene_version" : "8.8.0",
+       "minimum_wire_compatibility_version" : "6.8.0",
+       "minimum_index_compatibility_version" : "6.0.0-beta1"
+  },
+  "tagline" : "You Know, for Search"
+}
+```
+Now is time to install Kibana:
+```shell script
+$ helm install kibana elastic/kibana
+NAME:   kibana
+LAST DEPLOYED: Wed Apr 14 12:52:21 2021
+NAMESPACE: dcd
+STATUS: DEPLOYED
+```
+make sure kibana is deployed and then setup port forwarding for kibana:
+```shell script
+$ kubectl get deployment
+NAME                               READY   UP-TO-DATE   AVAILABLE   AGE
+helm-operator                      1/1     1            1           23d
+ingress-nginx-release-controller   1/1     1            1           82d
+kibana-kibana                      1/1     1            1           125m
+
+$ kubectl port-forward deployment/kibana-kibana 5601
+```
+Now kibana is accessible in the browser by: http://localhost:5601. To visualise the logs you need to create an index pattern and then see the data in the discovery part. To create the index pattern go to `Management` → `Stack Management` and then select `Kibana` → `Index Patterns`. 
+
+## EFK in EKS using managed AWS Elasticsearch
+
+In this method elastic search will deploy out of k8s clusters. For this purpose we use fluent bit instead of fluentd which is used for locally deployment for EFK. 
```

Comment:
`elasticsearch` will deploy as a managed service and it is outside of the kubernetes cluster.